--- core/server/data/schema/commands.js.old     2021-11-02 09:42:48.862517114 +0000
+++ core/server/data/schema/commands.js 2021-11-02 09:43:51.095300590 +0000
@@ -312,11 +312,13 @@
  */
 function createTable(table, transaction, tableSpec = schema[table]) {
     return (transaction || db.knex).schema.hasTable(table)
-        .then(function (exists) {
+        .then(async function (exists) {
             if (exists) {
                 return;
             }

+           await (transaction || db.knex).raw('SET sql_require_primary_key=0');
+
             return (transaction || db.knex).schema.createTable(table, function (t) {
                 Object.keys(tableSpec)
                     .filter(column => !(column.startsWith('@@')))
